### PR TITLE
Add support for doctrine/data-fixtures 2.0

### PR DESCRIPTION
**Pull request by mbabker**
As advertised

~~MongoDB ODM issue reported upstream - https://github.com/doctrine/data-fixtures/issues/504~~ Issue fixed

### CODE REVIEW

**Comment by mbabker** [comment]
This is ready to go with the upstream issue resolved.

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -35,6 +35,8 @@ jobs:
             symfony-version: "^7.1"
           - php-version: 8.4
             symfony-version: "^7.1"
+          - php-version: 8.4
```
Comment (alexisLefebvre) [inline .github/workflows/tests.yml:36]: Please remove this and update `7.1` to `7.2` on the previous lines.

Comment (mbabker) [reply .github/workflows/tests.yml:36]: Updated

**Comment by alexislefebvre** [comment]
Thanks! See the last release: https://github.com/liip/LiipTestFixturesBundle/releases/tag/3.2.0

**Pull request merged**